MAUI/Scheduler/resource-view.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: post
+title: Resource in MAUI Scheduler control | Syncfusion
+description:  Learn here all about grouping the appointments based on the resources in the day, week, and timeline views in Syncfusion WinUI Scheduler (SfScheduler) control.
```

Comment:
title: Resource view in .NET MAUI Scheduler control | Syncfusion

description: Learn here all about grouping the appointments based on the resources in the timeline day, timeline week,  timeline workweek and timeline month views in Syncfusion .NET MAUI Scheduler (SfScheduler) control.